bloom-housing/bloom
# fix: cannot save custom mailing, dropoff, or pickup address

# Pull Request Template

## Issue Overview

This PR addresses #2018


## Description

Could not save custom mailing, dropoff, or pickup address in LM.

## How Can This Be Tested/Reviewed?

Attempt to save mailing, dropoff, and pickup address of all types, ensure when you re-open the listing the address section is in the proper state.

I believe @YazeedLoonat is working on a followup PR for the LM tests that iterates on radio states in LM.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
